pages/p/[id].tsx

```diff
@@ -56,10 +56,9 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 const Post: React.FC<PropObj> = (props) => {
     let title = props?.post?.title
     let name = props?.post?.owner?.name
-
     if(props?.code) {
       return <Error statusCode={404} title="Post does not exist"/>
-    } else if(!props?.session?.user) {
+    } else if(!props?.session?.user || process.env.NEXT_PUBLIC_USE_TEST === 'false') {
```

Comment:
I wonder if you could just pass a test session into the props so that you don't have to use the flag here.